### DISCUSSION TITLE
finetune model

### DISCUSSION THREAD

**Discussion by drewskidang** [khoj-ai/khoj#399] [Q&A]
is it possible to use my own finetune llma with this

**Comment by debanjum** [comment]
Your fine tune of Llama/Llama 2? Khoj doesn't support swapping the offline models in any neat way.
But if you're fine with a very hacky solution, you could place any GPT4All compatible model at `~/.cache/gpt4all/llama-2-7b-chat.ggmlv3.q4_K_S.bin` and see if that works?

**Reply by Ellen7ions** [reply]
I propose a new PR #402 that supports specifying gpt4all models based on user input. I hope it helps @drewskidang @debanjum

**Comment by debanjum** [comment]
I'll convert this into a Github discussion, just in case you want to continue this conversation there. Hope that's fine

**Comment by Ellen7ions** [comment]
#402 might help you 😊

**Comment by drewskidang** [comment]
Appreciate it!!
